community/tw/elixirTW/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "elixirTW",
+  "title": "Elixir Taiwan @ Taipei",
+  "description": "We are a handful of elixir enthusiast based in greater Taipei area, come geek out with us !",
```

Comment:
Let's add more description.
I referred to https://www.meetup.com/elixirtw-taipei/ and helped you add this.
Thanks